modules/ROOT/pages/6.1.2-release-notes.adoc

```diff
@@ -0,0 +1,103 @@
+= TinyMCE 6.1.2
+:navtitle: TinyMCE 6.1.2
+:description: Release notes for TinyMCE 6.1.2
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.1.2 was released for {enterpriseversion} and {cloudname} on Wednesday, August 3^rd^, 2022. It includes {productname} 6.1.2. These release notes provide an overview of the changes for {productname} 6.1.2:
+
+* xref:bug-fixes[Bug fixes]
+* xref:known-issues[Known issues]
+
+[[bug-fixes]]
+== Bug fixes
+
+
+=== Invalid special elements were not cleaned up correctly during sanitization
+
+In {productname} 5.x, loading data that contained invalid special elements resulted in the invalid elements and their contents being removed before the data was placed in a {productname} document.
+
+In {productname} 6, a change to the DomParser introduced a regression: invalid elements were still filtered out, but their contents were placed in the {productname} document as plain text.
+
+The {productname} 6.1.2 update reverts this regression. When input data is being parsed by {productname} 6.1.2, invalid elements *and* their contents are both removed, as happened previously and is expected.
+
+
+=== An exception was thrown when deleting all content if the start or end of the document had a `contenteditable="false"`
+
+If the first or last element in a {productname} document had a `contenteditable="false"` property and the entire {productname} document was selected (eg, *Edit > Select All* was chosen), pressing the Delete or Backspace key resulted in a console error: `Uncaught Error: Node cannot be null or undefined`.
+
+With this update, the exception is no longer thrown in this circumstance. In {productname} 6.1.2, selecting all of a {productname} document that includes a `contenteditable="false"` property at the beginning or end of the document and deleting the selected contents produces no console error.
```

Comment:
Just wondering if saying 'element' rather than 'property' make a bit more sense for the sentence. No strong opinion though.
```suggestion
With this update, the exception is no longer thrown in this circumstance. In {productname} 6.1.2, selecting all of a {productname} document that includes a `contenteditable="false"` element at the beginning or end of the document and deleting the selected contents produces no console error.
```